lang/cgilua/Makefile

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2016 OpenWrt.org
```

Comment:
Actually, according to [this](https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md) I *can* use OpenWrt.